### DISCUSSION TITLE
Log file for imputation map-kmers and find-paths steps

### DISCUSSION THREAD

**Discussion by jesse-hill** [maize-genetics/phg_v2#227] [Ideas]
I'm running imputation on my reads and have gotten to the map-kmers and find-paths steps. A good number of my reads find no matching haplotype so I get the warning "No haplotypes found for read. No haplotype will be counted." Could there be an additional flag within map-kmers and find-paths to output all these warnings to a file, or at the very least incorporate a warning in the documentation to pipe this output to a file before running it? Because the printing to screen is a bit excessive.

**Comment by zrm22** [comment]
Hello,

This message will be removed once we merge in the kmer-mapping-diagnostic branch.  We are currently investigating poor mapping performance and have cleaned up a good deal of the excessive logging in the process.

We are hoping that we can get the imputation fixed up in the next few weeks.

Thank you for your patience.

**Comment by zrm22** [comment]
I have merged in the portion of code which should remove all of these excessive logging messages.  Please try rerunning the command using the newest release.
